### ISSUE TITLE
Deprecated ReflectionType function alert

### ISSUE THREAD

**Issue by joantp** [atom/atom#21759]
I often get a list of error popups which prevent me to work fluently. 

![Screenshot_20201125_123515](https://user-images.githubusercontent.com/18353260/100223170-6af51080-2f1b-11eb-8021-ddff8df5dab6.png)

Is thee a way to get rid of "Function ReflectionType::__toString() is deprecated" warning?

**Comment by darangi** [comment]
Does this happen when working with PHP files?

**Comment by joantp** [comment]
> Does this happen when working with PHP files?

Yes, it does.

**Comment by darangi** [comment]
Hi @joantp  do you still experience this?

**Comment by joantp** [comment]
Yes, I do. Shouldn't I? Anything new?

**Comment by Aerijo** [comment]
It looks to be caused by a package, are you able to reproduce this in safe mode?

In particular a package that does PHP analysis, like `ide-php`, is the likely cause. No core packages do language specific analysis like that.

**Comment by joantp** [comment]
> It looks to be caused by a package, are you able to reproduce this in safe mode?
> 
> In particular a package that does PHP analysis, like `ide-php`, is the likely cause. No core packages do language specific analysis like that.

Yes, you are right. Can not reproduce in safe mode, so it is not an issue of atom's core packages.

But the problem persists in normal mode and I can't guess what is the package causing this annoying problem.
